Bits Code: custom agent skills, GitLab scope warning (and more!) (#38273)
DataDog/documentation: content/en/bits_ai/bits_code/setup.md

1. Install the [GitLab Source Code integration][1]. For full installation and configuration steps, see the [GitLab Source Code integration guide][2].
1. Verify that the GitLab [service account][3] meets the following requirements:
   - The service account must have the [Developer role][4] on the project. This role can be inherited from a [group][5].
   - The service account's [personal access token][7] must have the following [scopes][6]: `api`, `write_repository`, and `read_user`. 

>    <div class="alert alert-warning">It's not possible to modify the scopes of an existing GitLab personal access token. If you need to create a token that includes the ones above, be sure to add <a href="/integrations/gitlab-source-code/#required-gitlab-scopes">all required scopes</a> needed by other Datadog products using the GitLab Source Code integration.</div>

Reviewer: ```suggestion
   <div class="alert alert-warning">You can't modify the scopes of an existing GitLab personal access token, so if you need a token that includes the scopes above, add <a href="/integrations/gitlab-source-code/#required-gitlab-scopes">all scopes required</a> by other Datadog products that use the GitLab Source Code integration.
```

A little more direct (feel free to tweak further)

Author: nice clean-ups! tweaked a lil more in https://github.com/DataDog/documentation/pull/38273/changes/1ede212534664dbc2a10d4a14a020122960f1632
